ansible/roles/pre-ansible/tasks/coreos_bootstrap.yml

```diff
@@ -0,0 +1,36 @@
+- name: CoreOS | Create bootstrap directory
+  raw: mkdir -p {{ bootstrap_script_dir }}
+
+- name: CoreOS | Check if we need to download pypy source
```

Comment:
Remove this check for now. We may decide to check for bins in the future. It would be helpful if you could add pypy to the gem-master. You can follow the kube_download_dir design pattern from here: https://github.com/danehans/ansible-role-ansible-controller